templates/beta-post.adoc

```diff
@@ -0,0 +1,186 @@
+---
+layout: post
+title: "TITLE"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/GITHUB_USERNAME
+author_github: https://github.com/GITHUB_USERNAME
+seo-title: TITLE - makes sure it ends with - OpenLiberty.io
+seo-description: DESCRIPTION
+blog_description: DESCRIPTION
+---
+= TITLE
+AUTHOR_NAME <https://github.com/GITHUB_USERNAME>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+// // // // // // // //
+// Above:
+// Do not insert any blank lines between any of the lines above.
+// Do not remove or edit the variables on the lines beneath the author name.
+//
+// Replace TITLE with the blog post title eg: MicroProfile 3.3 is now available on Open Liberty 20.0.0.4
+// Replace GITHUB_USERNAME with your GitHub username eg: lauracowen
+// Replace DESCRIPTION with a short summary (~60 words) of the release (a more succinct version of the first paragraph of the post).
+// Replace AUTHOR_NAME with your name as you'd like it to be displayed, eg: Laura Cowen
+// // // // // // // //
+
+BETA_RELEASE_SUMMARY
+
+// // // // // // // //
+// Change the RELEASE_SUMMARY to a quick sentence intro. Make sure to keep the blank lines 
+// // // // // // // //
+We have two beta packages for link:{url-about}[Open Liberty]:
+
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
+* <<allbeta, All Beta Features>>: a new, larger package that contains all Open Liberty beta and GA features and functions.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools (either the Jakarta EE 9 Beta Features package or the All Beta Features package, depending on what you want to try).
+
+If you try either package, <<feedback, let us know what you think>>.
+[#jakarta]
+== Jakarta EE 9 Beta Features package
```

Comment:
Switch these two sections round and change the introductory text of each to match what we had in the 20.0.0.10-beta post. Then make sure the instructional comments still make sense.

Reply:
Done